src/inc/AppxBundleManifest.hpp

```diff
@@ -143,6 +146,7 @@ namespace MSIX {
         std::uint64_t m_size;
         std::uint64_t m_offset;
         std::vector<Bcp47Tag> m_languages;
+        std::vector<std::string> m_scales;

```

Comment:
>std::string [](start = 20, length = 11)

why a string? can it be just UINT32?

Reply:
That just happened to be the default value when getting the attribute value when calling resourceNode->GetAttributeValue(). 

I can change to UINT32.